pwa/app/routes/apidocs.tsx

```diff
@@ -0,0 +1,304 @@
+                <Link to="/contact">contact</Link> page.
+              </p>
+            </li>
+            <li className="list-disc">
+              <p>
+                Found a bug or have a feature request? File an{' '}
+                <a href="https://github.com/the-blue-alliance/the-blue-alliance/issues/new">
+                  issue on GitHub.
+                </a>{' '}
+              </p>
+            </li>
+            <li className="list-disc">
+              <p>
+                Want to contribute?{' '}
+                <a href="https://github.com/the-blue-alliance/the-blue-alliance">
+                  Check out our code and send us a pull request!
+                </a>{' '}
+              </p>
+            </li>
+          </ul>
+        </div>
+
+        <div>
+          <h3 className="text-2xl">Getting Started</h3>
+          <p>
+            Before you get started using The Blue Alliance APIs, you need to be
+            familiar with a few elements of web technology. The Blue Alliance
+            APIs work by having your computer send a web request to our servers
+            asking for some piece of data, and our servers send the data back to
+            your computer. You can ask for information about teams or matches,
+            or even send us information, like letting us know there&apos;s a
+            robot photo we should add to our data set.
+          </p>
+          <p>
+            First, you need some way of sending HTTPS Requests to The Blue
+            Alliance&apos;s servers. This will be your primary means of
+            communication with TBA. For testing purposes, your web browser may
+            suffice. For more advanced applications, you may want to use an
+            external library, such as{' '}
+            <a href="https://www.w3schools.com/jquery/jquery_ajax_intro.asp">
+              jQuery
+            </a>{' '}
+            (Javascript),{' '}
+            <a href="http://docs.python-requests.org/en/master/">Requests</a>{' '}
+            (Python), or <a href="https://square.github.io/okhttp/">OkHttp</a>{' '}
+            (Java), or others.
+          </p>
+          <p>
+            You will also need to be familiar with{' '}
+            <a href="https://en.wikipedia.org/wiki/Json">JSON</a>, a
+            machine-readable format for sending and receiving data. Most of the
+            TBA APIs use JSON-formatted data, so you should find out how to
+            parse JSON text in the language of your choice.
+          </p>
+          <p>
+            Once you&apos;ve figured out how to make HTTPS requests, you will
+            need to figure out how to manipulate request and response headers.
+            These will be used to pass authentication keys to TBA and understand
+            the cache life of returned data.
+          </p>
+        </div>
+
+        <div>
+          <div>
+            <h3 className="text-2xl">
+              <Link to="/api-docs/apiread">Read API (v3)</Link>
```

Comment:
This link should be `/apidocs/v3`. can you make sure all the links match the original